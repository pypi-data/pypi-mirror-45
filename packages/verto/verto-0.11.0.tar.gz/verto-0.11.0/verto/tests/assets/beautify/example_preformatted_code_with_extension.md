Here is a fragment of code that is important:
```
string = "Hello World!"
```
