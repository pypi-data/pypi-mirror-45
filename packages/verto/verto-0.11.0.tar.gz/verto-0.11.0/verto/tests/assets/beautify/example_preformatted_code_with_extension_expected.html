<p>
 Here is a fragment of code that is important:
</p>
<pre><code>string = "Hello World!"
</code></pre>
